README.md

```diff
@@ -16,8 +39,13 @@ In this project, we hope to build a set of reference applications that make full
 
```

Comment:
[sw-precache demo](https://jeffy.info/sw-precache/) - Demo of the [`sw-precache` library](https://github.com/jeffposnick/sw-precache), which integrates with your site's build process to make it offline-first.